flowlogd/put_flow_logs.py

```diff
@@ -130,3 +130,20 @@ def get_log_enable_account_ids():
 
     res = jclient.vpc.describe_flow_log_enable_accounts('describe-flow-log-enable-accounts')
     return res['DescribeFlowLogEnableAccountsResponse']['accountIds']['item']
+
+def delete_flows_objects(bucket_name):
```

Comment:
Please change it like

def delete_flows_objects(acc):
    acc_id = acc['projectId']
    bucket_name = acc['bucketName']